portfolio/src/main/webapp/index.html

```diff
@@ -12,10 +12,12 @@
   <body>
     <div id="header"></div>
     <div id="splash-tiles"></div>
+      <form id="log-in-out">
+      </form>
     <form action="/data" method="POST">
       <label for="comment-preface">Leave a comment!</label><br>
       <input type="text" id="comment" name="comment"></input><br> 
-      <input type="submit" class="submit"></input>
+      <input type="submit" class="submit" id="submit-comment"></input>
     </form>
     <form id="/numberComments" method="GET">
```

Comment:
Do you show the email of the user that posted each comment?

Reply:
Just added.